### DISCUSSION TITLE
Colour Profiles

### DISCUSSION THREAD

**Discussion by p-taylor** [upscayl/upscayl#560] [Q&A]
Does upscayl honour embedded Colour Profiles or does it strip them out / ignore them during the process?

**Comment by aaronliu0130** [comment]
I... don't know, it's handled by REAL-ESRGAN. Perhaps you could do a test?

**Comment by p-taylor** [comment]
I have tested it now (had to find a mac with a compatible GPU). Unfortunately it converts to a generic sRGB profile.
This caused a colour shift in the images i tested which is not ideal if you care about colour consistency.

**Reply by aaronliu0130** [reply]
I have created a formal feature request from this discussion.

**Reply by amofjam** [reply]
Well darn - it strips the embedded color profile I use. I calibrate monitor; color is imperative. Hope this can get fixed soon! This is too awesome.
